tests/testing_utils.py

```diff
@@ -26,37 +26,55 @@ def make_initialize_job(model_json):
     )
     return initialize_job
 
-def make_hyperparams():
+def make_transform_split_job( \
+    model_json, raw_filepath, transform_function = lambda x: x):
+    transform_split_job = DMLJob(
+        JobTypes.JOB_TRANSFORM_SPLIT.name,
+        model_json,
+        "keras",
+        hyperparams=make_hyperparams(),
+        label_column_name='label',
+        raw_filepath=raw_filepath,
+        transform_function=transform_function
+    )
+    return transform_split_job
+
+def make_hyperparams(split=0.004):
```

Comment:
why is split a 'hyperparameter'? Hyperparameters are things that are adjusted in hyperparameter search/sweep, i.e. batch size, learning rate, epochs

Reply:
Not my design decision, you should ask @georgymh.